app/controllers/home_controller.rb

```diff
@@ -95,10 +95,10 @@ def dashboard
     @comments = @comments.group('DATE(FROM_UNIXTIME(timestamp))') if Rails.env == "production"
     @answer_comments = DrupalComment.joins(:answer, :drupal_users)
                              .order('timestamp DESC')
-                             .where('timestamp - answers.created_at > ?', 0)
+                             .where('timestamp - answers.created_at > ?', 86400)
```

Comment:
Just curious, what's this value? Could you add a comment at the end of this line, explaining? Tnx!